index.html

```diff
@@ -59,9 +59,162 @@ <h1>
 				</a>
 			</li>
 		</ul>
+		<a>
+			<img src="./img/icon/readmore.svg" alt="read more button">
+		</a>
+	</section>
+	<section id="my-works" >
+		<div class="works">
+			<h1>My Recent Works</h1>
+			<div class="work-card">
+				<div class="work-thumbnail">
+					<img  alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages" >
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
```

Comment:
- [ ] See the spaces around your technologies, and consider fixing them to match the design on the left

<img width="748" alt="3" src="https://user-images.githubusercontent.com/77843050/212496998-d65b00b8-fb21-449c-a959-e4f4763b9266.png">